chromium/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "chromium",
+  "version": "1.0.0",
```

Comment:
`"version": "2017.9.12"`

Reply:
Thank you. Don't worry about reviewing this too hard yet, it is still a WIP